src/server/game/Entities/Totem/Totem.h

```diff
@@ -27,7 +27,12 @@ enum TotemType
     TOTEM_STATUE     = 2 // copied straight from moongose, may need more implementation to work
 };
 // Some Totems cast spells that are not in creature DB
-#define SENTRY_TOTEM_SPELLID  6495
+
+enum TotemSpellIds : uint32
+{
+    SentryTotemSpell = 6495,
```

Comment:
@Kitzunu is the case here correct? I don't remember what format is used for this kind of enum

Reply:
it works, but using enum class would be safer

Reply:
enum class TotemSpellIds: uint32

Reply:
https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum